### PR TITLE
Date picker component

### PR DESCRIPTION
**Pull request by dgrubelic**
New date picker component PR.

### CODE REVIEW

**Comment by dgrubelic** [comment]
**Don't merge yet!** I still have some minor fix (bug reported on my repo) to push to this PR.

**Comment by sgomes** [comment]
Hey @dgrubelic, just wanted to give a quick heads-up that I haven't forgotten about your PR :)
I still plan to take an extensive look at this across multiple devices and browsers, but I haven't had the chance to do so yet.

Looking at your last comment, though, please let me know if you'd rather I wait a little before taking that look!

**Comment by dgrubelic** [comment]
Thanks :) This actually is really a minor bug fix. It will be pushed to PR tonight as soon as i get home. I'll notify you when it's available.

**Comment by dgrubelic** [comment]
@sgomes Fix pushed. Feel free to test things out. :)

**Comment by leifoolsen** [comment]
A date picker / Calendar component would be great to have in V2. Moreover, if you makes the component WAI-ARIA friendly, it will increase the usability. See e.g. https://www.w3.org/TR/wai-aria-practices-1.1/#datepicker

**Comment by dgrubelic** [comment]
Thanks for addressing this issue. I'll take care of it as soon as i can.

Guess like "dont-merge-yet" label is still active. :)

**Comment by dgrubelic** [comment]
Issue related to accessibility improvements dgrubelic/material-design-lite#25

**Comment by vibhoo** [comment]
Is PR is merged

**Comment by dgrubelic** [comment]
No, not yet. I still have few more things to implement but didn't have time over the past few weeks. I'll try to do it by the end of this week.

**Comment by nfavaron** [comment]
Hi, any update on this PR ? It's a really interesting feature :)

File: src/pickers/demo.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    </div>
+    <main class="mdl-layout__content">
+      <div class="page-content">        
+        <div class="mdl-grid">
+          <div class="mdl-cell">
+            <div><a name="attached"></a></div>
+            <h6>Datepicker attached to input</h6>
+            
+            <div id="demo-picker" class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label mdl-datepicker mdl-js-datepicker">
+              <input type="text" class="mdl-textfield__input mdl-datepicker__input" required>
```
Comment (rianby64) [inline src/pickers/demo.html]: I'm curious... what about if ```type="date"```? By default the ```input``` will render its default format _"mm/dd/yyyy"_ and the final result, I think, will be [like this](http://puu.sh/oY6kB/18c13266a2.png)...

**Comment by fwindolf** [comment]
Hi, any updates on when it will be merged?

**Comment by sgomes** [comment]
@dgrubelic Due to the big changes happening in v2 and the repo in general, I'm closing this PR for now. That said, this will likely form the basis of the implementation we do end up shipping with, when we do have the time to look at it. If you happen to have time yourself before then, please chat me up and I'll give you a guided tour of the current state of master at that point :)

**Pull request closed without merging**